src/relay/backend/contrib/dnnl/codegen.cc

```diff
@@ -392,22 +381,46 @@ class DNNLModuleCodegen : public CSourceModuleCodegenBase {
     code_stream_ << "using namespace tvm::runtime::contrib;\n";
     code_stream_ << "\n";
 
+    String func_symbol("all");
```

Comment:
I think it is okay that this is code a bit redundant to the one in c_codegen, because we are planning to remove this in the followup PRs and replace it with a json runtime.